cmd/notation/verify.go

```diff
@@ -246,12 +248,17 @@ func getVerifier(ctx context.Context) (notation.Verifier, error) {
 	}
 	fileCache, err := crl.NewFileCache(cacheRoot)
 	if err != nil {
-		return nil, err
+		// discard NewFileCache error directly as cache is not critical
+		logger.Debugf("failed to create file cache: %v", err)
```

Comment:
Since this is Notation CLI, we always want the crl file cache. IMO, we should keep failing on creating a new cache. 
By discarding cache error, we are actually discarding any error related to cache 'Get' and 'Set'.

Reply:
If the cache directory has a permission error, then the `NewFileCache` will return an error and failed the verification, which is not expected.

Reply:
Then a warning needs to be printed out to stderr instead of the debug log.
(Note: the error msg already includes `failed to create crl file cache`, so you don't have to repeat it in the msg here.)

Reply:
Updated.